cpu/cortexm_common/cortexm_init.c

```diff
@@ -46,7 +46,7 @@ void cortexm_init(void)
     /* set SVC interrupt to same priority as the rest */
     NVIC_SetPriority(SVCall_IRQn, CPU_DEFAULT_IRQ_PRIO);
     /* initialize all vendor specific interrupts with the same value */
-    for (int i = 0; i < CPU_IRQ_NUMOF; i++) {
+    for (int i = 0; i < (int)CPU_IRQ_NUMOF; i++) {
```

Comment:
The vendor specific interrupts start at number 0, which is what is being initialized here, but there are 16 core interrupts which have negative indices.